src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx

```diff
@@ -0,0 +1,276 @@
+        size="default"
+        title={<CardTitle value="Theories of Consciousness" />}
+        extra={
+          <a
+            className={styles.RemoveCardSupported}
+            onClick={() => RemoveCardDelegatedSupportedCamps()}
+          >
+            <CloseCircleOutlined /> Remove support{" "}
+          </a>
+        }
+        style={{ width: 760, marginBottom: 16 }}
+      >
+        <div>
+          <Row>
+            <Col span={12}>
+              <div className={styles.line_height}>
+                <p>
+                  Support delegated to:
+                  <span className={styles.Bluecolor}>Pranav</span>
+                </p>
+                <p>
+                  Nick Name:
+                  <span className={styles.Bluecolor}>rohit_telentelgia</span>
+                </p>
+              </div>
+            </Col>
+            <div></div>
+            <Col span={12} className={styles.border_left}>
+              <div className={styles.line_height1}>
+                <p>
+                  <b>Current Supported Camps:</b>
+                </p>
+                <p>
+                  1. &nbsp;
+                  <span className={styles.Bluecolor}>
+                    Technological Improvement
+                  </span>
+                </p>
+                <p>
+                  2. &nbsp;
+                  <span className={styles.Bluecolor}>Digital Identity</span>
+                </p>
+                <p>
+                  3. &nbsp;<span className={styles.Bluecolor}>Prototype</span>
+                </p>
+              </div>
+              <a className={styles.mrgn_left} onClick={showViewMoreModal}>
+                View More
+              </a>
+            </Col>
+          </Row>
+        </div>
+      </Card>
+      <Card
+        className={styles.cardBox_tags}
+        type="inner"
+        size="default"
+        title={<CardTitle value="Front End Language" />}
+        extra={
+          <a
+            className={styles.RemoveCardSupported}
+            onClick={() => RemoveCardDelegatedSupportedCamps()}
+          >
+            <CloseCircleOutlined /> Remove support{" "}
+          </a>
+        }
+        style={{ width: 760, marginBottom: 16 }}
+      >
+        <div>
+          <Row>
+            <Col span={12}>
+              <div className={styles.line_height}>
+                <p>
+                  Support delegated to:
+                  <span className={styles.Bluecolor}>Pranav</span>
+                </p>
+                <p>
+                  Nick Name:
+                  <span className={styles.Bluecolor}>rohit_telentelgia</span>
+                </p>
+              </div>
+            </Col>
+            <div></div>
+            <Col span={12}>
+              <div className={styles.line_height1}>
+                <p>
+                  <b>Current Supported Camps:</b>
+                </p>
+                <p>
+                  1. &nbsp;
+                  <span className={styles.Bluecolor}>
```

Comment:
I noticed this section is almost repeated. Can we iterate of create component and pass props?

Reply:
Done